### ISSUE TITLE
Script not running

### ISSUE THREAD

**Issue by pascalb09** [goabstract/abstract-sdk#186]
Hello,
i have tried to use abstract-sdk in the latest version yesterday version 7.xx, today 8.0.0-alpha.1 and i always get the following error when i call it:
./node_modules/abstract-sdk/dist/paginate.js
Module not found: Can't resolve 'core-js/modules/es7.symbol.async-iterator' in 'C:\dev\project\node_modules\abstract-sdk\dist'.

I tried to call it up with React :)

Do I have to install anything else besides the Abstract SDK?

**Comment by bitpshr** [comment]
Hi @pascalb09, thanks for reaching out. I verified that both `abstract-sdk@7.x.x` and `abstract-sdk@8.0.0-alpha.1` can be installed and that they build properly with a React application generated by `create-react-app@3.3.0`. The SDK installs its own copy of `core-js` as part of its dependency graph.

Could you share a little more information about your application's setup? Specifically, how are you building and running your application, and what does your `package.json` look like?

Thanks in advance for any additional information you can provide.

**Comment by weyert** [comment]
Yes, I am having the same problem. If you want to use this package as part of a Node LTS script it's not working looks like it's not meant to use standalone. You can reproduce it by doing the following:

```
const Abstract = require('abstract-sdk')

const client = new Abstract.Client({
  accessToken: process.env.DEMO_TOKEN,
})
```

**Comment by weyert** [comment]
I have made a private build using this change in the babel config which seems to work for me:

```json
"babel": {
    "presets": [
      "@babel/preset-flow",
      [
        "@babel/preset-env",
        {
          "corejs": 2,
          "useBuiltIns": "usage",
          "targets": {
            "node": 8
          }
        },
        "@babel/preset-react"
      ]
    ],
    "plugins": [
      "@babel/plugin-transform-runtime",
      "@babel/plugin-transform-async-to-generator",
      "@babel/plugin-proposal-object-rest-spread",
      "@babel/plugin-proposal-class-properties"
    ]
  }
```

**Comment by bitpshr** [comment]
Hi @pascalb09 and @weyert, thanks for adding this additional information.

After digging into this a bit more, `core-js/modules/es7.symbol.async-iterator` is only required since we're currently targeting Node 8 when building `abstract-sdk`. While `core-js` should still be installed when installing `abstract-sdk` via its dependency graph, we actually plan to [remove support for Node 8](https://github.com/goabstract/abstract-sdk/pull/189) and to instead target Node 10 in the coming days, as Node 8 will no longer be actively maintained and its official end of life date is 12/31/2019.

For now, please feel free to try `abstract-sdk@8.0.0-beta.1` using Node 10 or later until the upcoming `abstract-sdk@8.0.0` release, which should drop shortly after the new year.

**Comment by bitpshr** [comment]
Closing since this was fixed with #189. Thanks again @pascalb09 and @weyert, don't hesitate to reach out with any additional issues.
